crates/ruff_python_parser/src/lexer.rs

```diff
@@ -1491,19 +1489,39 @@ const fn is_ascii_identifier_start(c: char) -> bool {
 
 // Checks if the character c is a valid starting character as described
 // in https://docs.python.org/3/reference/lexical_analysis.html#identifiers
+#[cold]
 fn is_unicode_identifier_start(c: char) -> bool {
-    is_xid_start(c)
+    unicode_ident::is_xid_start(c)
 }
 
 // Checks if the character c is a valid continuation character as described
 // in https://docs.python.org/3/reference/lexical_analysis.html#identifiers
 fn is_identifier_continuation(c: char) -> bool {
-    match c {
-        'a'..='z' | 'A'..='Z' | '_' | '0'..='9' => true,
-        c => is_xid_continue(c),
+    if c.is_ascii() {
+        ASCII_CONTINUE.0[c as usize]
+    } else {
+        is_unicode_identifier_continue(c)
     }
 }
 
+#[cold]
+fn is_unicode_identifier_continue(c: char) -> bool {
+    unicode_ident::is_xid_continue(c)
+}
+
+const T: bool = true;
+const F: bool = false;
+
+#[repr(C, align(64))]
+struct Align64<T>(pub(crate) T);
```

Comment:
How come you're forcing 64-bit alignment here?

Reply:
(This is copied out from the `unicode_ident` crate just for quickly benchmarking.)